security-analysis/src/main/java/com/powsybl/security/Security.java

```diff
@@ -219,14 +332,105 @@ public static String printLimitsViolations(List<LimitViolation> violations, Limi
     }
 
     private static float getAbsValueLimit(LimitViolation violation) {
-        return Float.valueOf(Float.toString(violation.getLimit()) + (violation.getLimitReduction() != 1f ? " * " + violation.getLimitReduction() : ""));
+        return Math.abs(violation.getValue() - violation.getLimit() * violation.getLimitReduction());
```

Comment:
It's strange to use the limitReduction here... The header of the column is "abs(value-limit)" and this value could be misunderstood

Reply:
Done. I've fixed the code.